package.json

```diff
@@ -23,6 +23,7 @@
     "error-stack-parser": "^2.1.4",
     "glob": "^8.0.3",
     "minimist": "^1.2.6",
+    "same-object": "^1.0.0",
```

Comment:
You probably want to remove `deep-equal` as well?

Reply:
Thanks! Actually did but didn't push!